iphone/TitaniumKit/TitaniumKit/Sources/API/TiExceptionHandler.m

```diff
@@ -156,8 +156,19 @@ - (id)initWithDictionary:(NSDictionary *)dictionary
     if (_backtrace == nil) {
       _backtrace = [[[dictionary objectForKey:@"stack"] description] copy];
     }
-    _nativeStack = [[dictionary objectForKey:@"nativeStack"] copy];
     _dictionaryValue = [dictionary copy];
+
+    id nativeStackObject = [dictionary objectForKey:@"nativeStack"];
+    if (nativeStackObject) {
+      if ([nativeStackObject isKindOfClass:[NSArray class]]) {
+        nativeStackObject = [nativeStackObject copy];
```

Comment:
NO need to copy here. copy should be used with  _nativeStack = [nativeStackObject copy];